.changeset/two-turkeys-join.md

```diff
@@ -0,0 +1,5 @@
+---
+'react-textarea-autosize': patch
+---
+
+Force display: block for the hidden textarea to prevent site overrides
```

Comment:
```suggestion
Force `display: block` for the hidden textarea to prevent other styles from overriding it and thus breaking the resizing functionality
```